bricks/template/__brick__/{{project_name.snakeCase()}}/ios/fastlane/Fastfile

```diff
@@ -141,7 +142,7 @@ platform :ios do
   desc 'set build number with number of commits'
   private_lane :bump_build do
     increment_build_number(
-      build_number: number_of_commits,
+      build_number: ENV["GITHUB_RUN_NUMBER"],
```

Comment:
Should we define a new env in `Environments` to get the build number from ENV[]?

```suggestion
      build_number: Environments.BUILD_NUMBER,
```